common/locales/fr.json

```diff
@@ -169,7 +169,18 @@
     }
   },
   "global": {
-    "datetime": {}
+    "datetime": {
+      "dateLong": "dd D.MM.YYYY",
+      "dateShort": "dd D.M.",
+      "dateTimeLong": "dd D.MM.YYYY HH:mm",
```

Comment:
Is there a reason we use "dd L HH:mm" for all other languages but not for French? Similar for `dateLong` as well.

Reply:
I did it because the date format for french was not correct. I'll move this to a separate PR, where I introduce a customizable date format. Because one might want to use eCamp in English, but not in "football-is-mostly-played-with-hands" format ;) 
[#3217](https://github.com/ecamp/ecamp3/issues/3217)